yt/data_objects/tests/test_particle_trajectories.py

```diff
@@ -43,3 +47,22 @@ def test_etc_traj():
         def field_func(name):
             return traj[field]
         yield GenericArrayTest(ds, field_func, args=[field])
+
+def test_uniqueness():
+    n_particles = 2
+    n_steps = 2
+    ids = np.arange(n_particles, dtype=int) % (n_particles//2)
+    data = {'particle_index': ids}
+    fields = ['particle_position_x', 'particle_position_y', 'particle_position_z', 'particle_index']
+    negative = [False, False, False, False]
+    units = ['cm', 'cm', 'cm', '1']
+
+    ts = DatasetSeries([fake_particle_ds(fields=fields, negative=negative, units=units,
+                                         npart=n_particles, data=data)
+                        for i in range(n_steps)])
+    try:
+        traj = ts.particle_trajectories([0])
+    except YTIllDefinedParticleData:
+        pass
+    else:
+        assert False
```

Comment:
it might be cleaner to use `assert_raises` here

Reply:
Didn't know!